src/sentry/auth/access.py

```diff
@@ -293,6 +314,13 @@ def accessible_project_ids(self) -> FrozenSet[int]:
         """
         return self.project_ids_with_team_membership
 
+    def has_organization_role(
+        self, role: str, organization: Organization, user_id: int | None
+    ) -> bool:
+        if self._member:
+            return has_organization_role(self._member.user_id, role, organization)
```

Comment:
lets explicitly type out key-values
```
return has_organization_role(user_id=self._member.user_id, role_id=role, organization=organization)
```

Reply:
especially since the signature of this method looks like org is the second param and role is the 3rd 🤔